docs/docs/concepts/redirects.mdx

```diff
@@ -1,9 +1,143 @@
+utilising the SDK in Single Page Applications (ReactJs, Angular, VueJs etc.) or
+using redirects in Server Side Applications (NodeJs, PHP, Golang etc.).
+
+On this page of the documentation we will be covering Browser Redirects and how
+to configure it on the Ory Cloud console. If you are interested in utilising the
+SDK for Single Page Applications, check out our [SDK](./services-api.mdx)
+documentation.
+
+For a deeper dive into the background of this setup, head over to our open
+source documentation:
 
 - https://www.ory.sh/kratos/docs/concepts/browser-redirect-flow-completion
+
+## Allow list
+
+The Ory Cloud allows dynamic redirects to be set using the `?return_to=` query
+parameter on [Self-Service Flows](./self-service.mdx). As an example a user
+opens a sharable link to go to https://myapp.com/posts, however, this url
+requires the user to be signed in and thus redirects the user back to the login
+page. To return the user back to the original url we can append
+`?return_to=https://myapp.com/posts` to the specified Self-Service initialise
+flow url.
+
+The `Allow list` is used in this scenario to prevent Open Redirect Attacks by
+only allowing certain domains, or paths on a domain. As shown in this example
+below the https://sub.domain.myapp.com/only/path needs to match perfectly for
+this sub-domain and path for it to be allowed. Other redirects using myapp.com
+will fail, however, for https://anotherapp.com will succeed on any path.
+
+![Browser Redirects Allow List](../images/browser-redirects/browser-redirects-allow-list.png)
+
+## Redirect Flows
+
+The Ory Cloud has a total of 6 flows, Login, Registration, Verification,
+Recovery, Settings and Logout which can be configured to redirect back to any
+url you specify. As an example you can redirect the user to your application
+home screen after a logout or to a specific url on a sub-domain after a settings
+password update. On project creation the redirect flows are all set to the
+default Ory Cloud Managed UI. For more information on the Ory Managed UI, please
+see the [Managed User Interface](./managed-ui.mdx) documentation.
+
+### Login, Registration and Settings
+
+The Login and Registration flows have three fields (all optional):
+
+1. `Default Redirect Url` domain or path
+2. An `Post-Password Redirect Url`
+3. An `Post-OIDC Redirect Url` (**currently not supported**)
+
+And Settings has three fields (all optional):
+
+1. `Default Redirect Url` domain or path
+2. An `Post-Password Redirect Url`
+3. An `Post-Profile Redirect Url`
+
+:::info
+
+Setting `Post-OIDC` redirects for Login and Registration are currently not
+supported since an Ory Cloud project cannot be setup to use OIDC yet. This will
+be supported in the future. For more information see the
+[Ory Cloud Roadmap](../early-access.mdx).
+
+:::
+
+When setting the `Default Redirect Url` as a FQDN (fully qualified domain name)
+such as e.g. https://domain.example/login it will overwrite the
```

Comment:
nit: this is bit confusing cause first you use domain.example/login and then immediately to describe path overrides you use a different domain. Maybe let's use single domain within this particular example?